P14Nim/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿/*
+ Well... this didnt work xD
+ apparantly, Ryder (or however you spell it) console window with its mat, grey tone, is classified as black -,,-
+Console.BackgroundColor = ConsoleColor.Black;
+Console.Clear();
+*/
+
+Console.WriteLine("Hello and Welcome User!");
+Console.WriteLine("What is your name?");
+string name = Console.ReadLine();
+Console.Clear();
+Console.WriteLine("Hello " + name + "!");
+Console.WriteLine("I am The artificial intelligence Jianshizi.");
+Console.WriteLine("Which just so happens to also be the original version of the game we are gonna play today! NIM!");
+Console.WriteLine("Now, Let me explain the game for you.");
+Console.ReadKey();
+Console.Clear();
+Console.WriteLine("We have 20 matches total.");
+Console.WriteLine("Each round Either You, or I, draw one to three of them.");
+Console.WriteLine("The looser is the one whom draws the last match!");
+Console.WriteLine("You, my dear " + name + ", get to go first!");
+Console.ReadKey();
+Console.Clear();
+
+//basic information the "game" needs to be able to do what i want...
+//not sure i should be making comments like this... is there a point?
+int totalMatches = 20;
+int  currentPlayer = 1;
+int takenMatches;
+Random random = new Random();
+
+
+//The game!
+start:
+UpdateMatchDisplay(totalMatches);
+Console.WriteLine($"\nCurrent matches: {totalMatches}");
+
+for (int i = 0; i < totalMatches; i++)
+{
+    Console.Write("|");
+}
+Console.WriteLine();
```

Comment:
Then again, this seems almost redundant then, to have this, too, or?
You could put this into the `UpdateMatchDisplay` function.